Remove 'solar power' param from Comm Sat contracts

lpgagnon opened the pull request:
Because the requirement is just a 'gotcha'.
- All those contracts are gated behind First Scientific Satellites,
  so solar panels are necessarily available
- Without any specific power output needed, the requirement can
  be satisfied with a single tiny panel that doesn't actually provide
  enough power for anything

So the requirement doesn't increase contract difficulty in any way
other than punishing players who fail to notice it

siimav commented:
I'm happy to remove it from contracts other than *First Communications Satellite*. Maybe this one should still have the solar panel requirement since control of the sat isn't given over to a customer. Btw, the contract description also says to include a solar panel.

lpgagnon commented:
push -f: reverted change to First Comm Sat (I'm not convinced by the 'not given over to customer' argument, but don't care that much), and fixed the description that still talked about power generation. (and rebased to latest master)
